api/v1alpha1/openapi.yaml

```diff
@@ -4796,6 +4796,18 @@ components:
     Status:
       type: object
       properties:
+        apiVersion:
+          type: string
+          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources.'
+        kind:
+          type: string
+          description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds.'
+        metadata:
+          $ref: '#/components/schemas/ListMeta'
```

Comment:
What's this for?

Reply:
Copied that from K8s API. It optionally contains the `continue` and `remainingItemCounts` that exist lists / collections of resources. I _suspect_ this is because the `Status` is mainly used in responses to `deletecollections` calls that return lists, but I didn't manage to reproduce a case where K8s API would actually return a non-empty metadata. Remove?

Reply:
If we don't use it, I say remove

Reply:
Just to be clear, you mean removing only `metadata` in the Status object, right?
In the UI, `metadata` is used for lists pagination so it's needed in `DeviceList`, ` FleetList` etc